apps/jupyter/run-jupyter.sh

```diff
@@ -7,7 +7,13 @@ die() {
 
 mkdir -p $HOME/.jupyter
 if [[ -f /.dockerenv ]]; then
-    APP_DIR=/app
+    if [[ $(whoami) == "aryn" ]]; then
```

Comment:
This feels weird to need. Can we consolidate on just having APP_DIR=/app?
The only thing it seems to be used for is WORK_DIR and /app/work makes more sense anyway, although I'd probably drop that as well; no one uses the jupyter containers except in customer envs and we don't have persistent storage there anyway.